sdk/communication/Azure.Communication.Sms/README.md

```diff
@@ -36,6 +36,14 @@ string connectionString = "YOUR_CONNECTION_STRING"; // Find your Communication S
 SmsClient client = new SmsClient(connectionString);
 ```
 
+Alternatively, SMS clients can also be authenticated using a valid token.
```

Comment:
Should we tell users to have their client id, tenant and client secret as env variables for DefaultAzureCredential to work?

Reply:
Done, thanks for the suggestion Jorge!